charts/seed-bootstrap/dashboards/extensions-dashboard.json

```diff
@@ -185,10 +185,34 @@
         "align": false,
         "alignLevel": null
       }
+    },
+    {
+      "datasource": "loki",
+      "gridPos": {
+        "h": 11,
+        "w": 24,
+        "x": 0,
+        "y": 9
+      },
+      "id": 5,
+      "options": {
+        "showTime": true,
+        "sortOrder": "Descending"
+      },
+      "targets": [
+        {
+          "expr": "{container_name=~\"(gardener-extension)(.+)\"}",
```

Comment:
There is no way to filter which extensions are shown here. Could you add a filter in grafana so you can select an extension?

Reply:
Done